lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -533,19 +529,19 @@ def generate_test_result_tree(tmp_path: Path, dir_name: str) -> Path:
                 sub_dir / "subdir1" / "subdir14" / "subdir141" / "f1412_sym",
             )
             os.symlink(
-                Path("..") / "subdir141",
+                Path("../subdir141"),
                 sub_dir / "subdir1" / "subdir14" / "subdir141" / "f1413_sym",
             )
             os.symlink(
                 Path(".") / "f1411.txt",
```

Comment:
Is that really how we want to express this?  Wouldn't `Path("./f1411.txt")` or `Path("f1411.txt")` be better?